terraform/workspace_variables/dev.tfvars.json

```diff
@@ -2,5 +2,7 @@
   "environment_name": "dev",
   "key_vault_name": "s165d01-dqtnoti-dv-kv",
   "resource_group_name": "s165d01-dqtnoti-dv-rg",
-  "resource_prefix": "s165d01"
+  "resource_prefix": "s165d01",
+  "enable_blue_green": false,
+  "notify_storage_account_name": "s165d01dqtnotinotifydv"
```

Comment:
This name could be improved, calling it having `noti` in the name twice isn't helpful.  It should either be `s165d01dqtnotidv` or `s165d01dqtnotiappdv` (app since at the minute the storage is just being used to back the function app)

Reply:
fixed